src/views/Results.vue

```diff
@@ -869,4 +938,13 @@ export default {
 		}
 	}
 }
+
+.search-wrapper {
+	margin-top: calc(-1 * var(--default-grid-baseline));
+	margin-left: var(--default-grid-baseline);
```

Comment:
Let's change this to margin-inline-start/end and margin-block-start so that it also works in RTL mode. `auto` makes it aligned on the right and the end margin is needed because the header and the question sections are not of the same width.

```suggestion
	margin-block-start: calc(-1 * var(--default-grid-baseline));
	margin-inline-start: auto;
	margin-inline-end: var(--default-clickable-area);
```